dgp/core/controllers/dataset_controller.py

```diff
@@ -216,9 +188,12 @@ def dataframe(self) -> DataFrame:
             self._dataframe: DataFrame = concat([self.gravity, self.trajectory], axis=1, sort=True)
         return self._dataframe
 
-    def align(self):
+    def align(self):  # pragma: no cover
+        """
+        TODO: Utility of this is questionable, is it built into transform graphs?
```

Comment:
Let's chat about this today when you get a free minute.  Indeed, this may no longer be necessary here.